source/decisions/2023-09-25-support-djangocon-africa.rst

```diff
@@ -0,0 +1,18 @@
+2023-03-04 Support of DjangoCon Africa 2023
+===============================================
+
+Background
+----------
+
+DjangoCon Africa takes place on 6-11 of November 2023. On 7 September 2023 the DjangoCon
+Africa Chair contacted the trustees by email to request monetary support for the event.
+
+
+Resolution
+----------
+
+The decision was made to support DjangoCon Africa 2023 to a value equivalent to 
+Silver level sponsorship.
+
+The exact mechanism of doing so (donation, grant, sponsorship) is considered an 
+implementation detail to be determined by UKPATL.
```

Comment:
I don't consider this a UKPATL matter at all - purely a UKPA one - can you elaborate on where you're coming from please?

Reply:
I'd thought that the funds would be coming from UKPATL, but if that's not the case, happy to remove this bit altogether, given you other comment about implementation details.